packages/cli/src/rest.test.ts

```diff
@@ -75,6 +86,16 @@ describe('CLI rest', () => {
     expect(console.log).toBeCalledWith(expect.stringMatching('Patient'));
   });
 
+  test('Post command with empty body', async () => {
```

Comment:
adds test coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=2313&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Frest.ts&line=50